profile/README.md

```diff
@@ -18,3 +18,20 @@ You can learn more about this subproject on the [website](https://www.jenkins.io
 
 The GitHub organization of the main Jenkins community including repositories of the [Jenkins core](https://github.com/jenkinsci/jenkins), plugins, libraries and developer tools is here:
 https://github.com/jenkinsci/
+
+### Jenkins Contributor Awards 2023
+
+The annual Jenkins awards is happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
+
+Nomination categories are:
+- Most Valuable Jenkins Advocate → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6035)
+- Most Valuable Jenkins Contributor → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6033)
+- Jenkins Security MVP → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6034)
+
+Awards timeline:
+- Nominations close: Friday, March 3
+- Voting opens: Wednesday, March 8
```

Comment:
Voting opens: Wednesday, March 8 → [Vote Here](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)

Reply:
Suggest to add link to voting form

Reply:
Creating a dedicated PR for these proposed changes @alyssat